library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -85,6 +86,21 @@ void commitPlan(PlanInfo plan) {
     }
   }
 
+  void commitAddNamedPlan(PlanInfo planInfo) {
+    // remove first
+    Performer performer = getNamedPerformer(planInfo);
```

Comment:
This should be capable of returning null in the event that no performer is presently associated with this name.